config/canvas.php

```diff
@@ -15,6 +15,20 @@
 
     'path' => env('CANVAS_PATH_NAME', 'canvas'),
 
+    /*
+    |--------------------------------------------------------------------------
+    | User Model
+    |--------------------------------------------------------------------------
+    |
+    | This is the User model that Canvas will use to associate
+    | to the author of the post.
+    |
+    | Default: null or Illuminate\Foundation\Auth\User::class
+    |
+    */
+
+    'model' => null,
```

Comment:
Let's use `user` as the variable name here. I'm also thinking that instead of setting this as `null`, we just set it to `Illuminate\Foundation\Auth\User::class` right off the bat.